### ISSUE TITLE
hidden

### ISSUE THREAD

**Issue by brianteeman** [akeeba/panopticon#106]
What is it you are trying to do here - its not correct.

![image](https://github.com/akeeba/panopticon/assets/1296369/6484117a-8ca4-4c26-a8f4-89d7a0a2a36c)

**Comment by nikosdion** [comment]
Look the checkbox above which is wrapped in aria-hidden. Because of the way the helper works I cannot pass a different title for sighted users (#) and screen readers (eg Item Number). So I’m displaying the former with aria-hidden to take it out of the screen reader’s parsed DOM and use the latter as visually hidden so sighted users don’t get to see it. It’s essentially the same trick we do with icons which show the short icon to sighted users and the long description to screen readers.

**Comment by brianteeman** [comment]
It doesn't work as you expect. I will do a pr tomorrow

**Comment by brianteeman** [comment]
the bug to fix is that in this case you end up with two focussable elements

**Comment by brianteeman** [comment]
More info https://accessibilityinsights.io/info-examples/web/aria-hidden-focus/

> Users can still use the keyboard to navigate to any focusable child elements, but their content is inaccessible to people who use assistive technologies. For example, screen readers are silent. (An element is focusable if it can receive input focus via scripting, mouse interaction, or keyboard tabbing.)
